dataduct/database/database.py

```diff
@@ -61,6 +65,7 @@ def add_relation(self, relation):
 
         self._relations[relation.full_name] = relation
 
+    @property
```

Comment:
It felt like a very property-ish thing for me.

Especially since the function isn't prefixed with 'get' (i.e. get_relations)